docs/egg_model_example.rst

```diff
@@ -0,0 +1,149 @@
+==========================
+The Egg model case
+==========================
+
+
+The Egg model  is a synthetic reservoir model developed by `J. D. Jansen et al., 2013 <https://rmets.onlinelibrary.wiley.com/doi/pdf/10.1002/gdj3.21>`_ [1]. We used the Egg model in the Computational Geosciences group at SINTEF Digital as a test case for developing data-driven models and testing new calibration methods for history matching.
+
+.. image:: ./_static/sintef_logo.jpg
+   :scale: 10%
+
+
+Data files
+==========
+
+- `The Egg Model - data files  <https://data.4tu.nl/articles/The_Egg_Model_-_data_files/12707642>`_ [2]
+
+
+Modifications to the simulations setup
+======================================
+
+We have modified the Egg model input file `Egg_Model_ECL.DATA` in order to follows Flownet workflow.
+
+1. COMPAT section is changed to have differences in depth between injectors and producers. With this, we avoid problems related to degenerate tetrahedron that we obtain in the triangulation procedure for the network generation.
```

Comment:
```suggestion
1. The COMPAT section is changed to have differences in depth between injectors and producers. With this, we avoid problems related to degenerate tetrahedrons that we would otherwise obtain in the triangulation procedure for the network generation.
```